packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -9,6 +9,7 @@ import { BinaryLink } from 'App/Components/Routes';
 import getRoutesConfig from 'App/Constants/routes-config';
 import { currentLanguage, changeLanguage } from 'Utils/Language';
 import LiveChat from 'App/Components/Elements/LiveChat';
+import Text from '../../../../../../components/src/components/text/text';
```

Comment:
@maryiafrantsava-deriv Text component should be imported at line #3
```
import { Div100vhContainer, Icon, MobileDrawer, ToggleSwitch, Text } from '@deriv/components';
```

Reply:
i've done